tests/test_pages.py

```diff
@@ -50,10 +51,15 @@ def test_menu_item(selenium, user, url):
     password_field.send_keys(user.password)
     password_field.submit()
 
-    account_menu = WebDriverWait(selenium, 10).until(
-        EC.presence_of_element_located((By.ID, 'account_menu'))
-    )
-    assert account_menu.text == user.name, 'Logged in user shows the correct name'
+    try:
+        account_menu = WebDriverWait(selenium, 10).until(
+            EC.presence_of_element_located((By.ID, 'account_menu'))
```

Comment:
Should we be looking at the future and first try the new user nav and then fall back to Foreman 2.4 and older?

Reply:
That's a good idea.